### DISCUSSION TITLE
Is there an external command for categorizing formulae/casks?

### DISCUSSION THREAD

**Discussion by aaronliu0130** [Homebrew/discussions#3474] [Everyday usage]
Categorizing has been repeatedly declined, and I recently learned that homebrew supports external commands and brew cask started out as one. So is there an external command for categorizing formulae/casks? Bonus question: Is there a command to sort formulae/casks by number of downloads?

**Comment by SMillerDev** [comment]
https://formulae.brew.sh/analytics/install-on-request/365d/ has the latter

**Reply by aaronliu0130** [reply]
Is there one for casks?

**Reply by SMillerDev** [reply]
Yes, same website: https://formulae.brew.sh/analytics/cask-install/365d/
